db/changes.go

```diff
@@ -1054,7 +1075,10 @@ func (col *DatabaseCollectionWithUser) SimpleMultiChangesFeed(ctx context.Contex
 			if err != nil {
 				change := makeErrorEntry("User not found during reload - terminating changes feed")
 				base.DebugfCtx(ctx, base.KeyChanges, "User not found during reload - terminating changes feed with entry %+v", base.UD(change))
-				output <- &change
+				select {
```

Comment:
Same as elsewhere, I think checking ctx.Err() is preferable to a select.

Reply:
We have to use a select in the case that output consumer has exited. The options for things that can happen:

1. send to the feed works, the channel is unblocked
2. the send to the feed blocks, because the buffer is full. The buffer will empty because the consumer is slow to catch up
3. the send to the feed blocks, because the buffer is full. The context hasn't cancelled yet, but it will while the buffer is full.

Without this code, this go routine might not exit. In practice, this would be unlikely to happen with an `context.Err() != nil` check - the context would have to alive and then cancelled when the output is sending. This is either an extremely likely data race, or a circumstance where the output buffer is already full and the context is just slow to cancel. Given the buffer size is 50, it's not super likely that it is full.

I still think this is the right behavior to include a `<- context.Done()` when we do a send in order to not to hang a goroutine.